res/controllers/common-controller-scripts.js

```diff
@@ -245,6 +245,89 @@ script.midiDebug = function(channel, control, value, status, group) {
         " status: 0x" + status.toString(16) + " group: " + group);
 };
 
+
+
+class StringDeburr {
+    constructor() {
+        this.regexLatin = /[\xc0-\xd6\xd8-\xf6\xf8-\xff\u0100-\u017f]/g;
+
+        this.regexComboMark = /[\u0300-\u036f\ufe20-\ufe2f\u20d0-\u20ff]/g;
+
+        this.deburredLetters = {
+            // Latin-1 Supplement block.
+            "\xc0": "A",  "\xc1": "A", "\xc2": "A", "\xc3": "A", "\xc4": "A", "\xc5": "A",
+            "\xe0": "a",  "\xe1": "a", "\xe2": "a", "\xe3": "a", "\xe4": "a", "\xe5": "a",
```

Comment:
have you considered using a `sparse` array instead? Maybe that would result in a speedup because indexing could be cheaper?

Reply:
How do create a correspondance between UTF-8 chars and Latin-1 with sparse arrays? I don't get it.